scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/SchedulerServiceImpl.java

```diff
@@ -1362,6 +1361,31 @@ private int executeFunction(BiFunction<RestClient, Integer, InputStream> action,
         return success;
     }
 
+    private int executeVoidFunction(BiFunction<RestClient, Integer, Void> action, List<Integer> jobIdList,
+            String actionName) throws ServiceException, RestServerException {
+
+        RestClient restClientProxy = getRestClientProxy();
+
+        int failures = 0;
+        int success = 0;
+
+        for (Integer jobId : jobIdList) {
+            try {
+                action.apply(restClientProxy, jobId);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)